exporter/tanzuobservabilityexporter/transformer_test.go

```diff
@@ -223,6 +224,73 @@ func TestTraceStateTranslatedToTag(t *testing.T) {
 	assert.False(t, ok)
 }
 
+func TestSpanForSourceTag(t *testing.T) {
+	inNanos := int64(50000000)
+
+	//TestCase1: default value for source
+	att := pdata.NewAttributeMap()
```

Comment:
Can we give `att` a more descriptive name throughout this test? Like `resAttrs`?

Reply:
done